homeassistant/components/smartthings/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "smartthings",
   "name": "SmartThings",
+  "version": "0.0.1",
```

Comment:
This is not correct. This is a built-in integration. Those don't need (and should not have) a version number. Version numbers for built-in integrations are automatically inherited from the core version number.

../Frenck